utils/docker/vcluster/daos-cm.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
```

Comment:
As stated elsewhere file permissions are wrong here.

Reply:
If I understand, you want to make it executable ?
If yes, I would like to remove the ".sh" suffix. Is it OK for your.

Reply:
fixed with commit https://github.com/daos-stack/daos/pull/7556/commits/910df9b202e5577ca34d062a72984d8c1011b3e2